granulate_utils/linux/cgroups/memory_controller.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from abc import abstractmethod
+from pathlib import Path
+from typing import Optional, Union
+
+from granulate_utils.linux.cgroups.base_controller import BaseController
+from granulate_utils.linux.cgroups.cgroup import CgroupCore, CgroupCoreV1, CgroupCoreV2
+
+
+class MemoryControllerInterface:
+    memory_limit: str
+    usage: str
+    max_usage_in_bytes: str = ""
+    swap_limit: str = ""
```

Comment:
Why are these 2 initialized empty?

Reply:
I moved the initialization inside the subclasses instead.